### PR TITLE
get names consistent; associated doc updates based on test walkthrough

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees the cakephp-ex changes for issue3358

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ These are some special considerations you may need to keep in mind when running
...
 
 ###Installation: 
-These steps assume your OpenShift deployment has the default set of ImageStreams defined.  Instructions for installing the default ImageStreams are available [here](http://docs.openshift.org/latest/admin_guide/install/first_steps.html)
+These steps assume your OpenShift deployment has the default set of ImageStreams defined.  Instructions for installing the default ImageStreams are available [here](http://docs.openshift.org/latest/admin_guide/install/first_steps.html).  If you are defining the set of ImageStreams now, remember to pass in the proper credentials via the --config option.
```
Comment (bparees) [inline README.md]: proper cluster-admin credentials and to create the ImageStreams in the 'openshift' namespace

File: README.md [hunk 2]
```diff
@@ -23,40 +23,42 @@ These steps assume your OpenShift deployment has the default set of ImageStreams
 
 		$ oc new-app openshift/templates/cakephp.json -p SOURCE_REPOSITORY_URL=<your repository location>
 
-4. Watch your build progress  
+4. Depending on the state of your system, and whether additional items need to be downloaded, it may take around a minute for your build to be started automatically.  If you do not want to wait, run
+
+   	     	$ oc start-build cakephp-example
```
Comment (bparees) [inline README.md]: this is rendering with an extra indentation inside the code block (probably an issue in your PR for the other repos too).  guessing it's a tabs vs spaces issue.  use all spaces or two tabs.

Comment (gabemontero) [inline README.md]: yep, see it now ... addressed here and where present in the other files

**Comment by gabemontero** [comment]
@bparees I *think* the commit/squash/push for responses to you comments has been done with sufficient correctness, but am not 100% sure ... take peek when you can and let me know ... based on the results, I'll move onto the other examples (and hopefully nail down this aspect of using git/pull reqs)

**Comment by bparees** [comment]
lgtm.

**Pull request merged**